lab_Arrays/src/main/java/comper/Europa.java

```diff
@@ -0,0 +1,32 @@
+package comper;
```

Comment:
comper -> compare  - pay attention for typos and fix it, IDEA should highlight typos in words so do not skip those warnings